lib/ansible/modules/monitoring/zabbix/zabbix_screen.py

```diff
@@ -149,6 +153,25 @@
         graph_width: 200
         graph_height: 100
   when: inventory_hostname==groups['group_name'][0]
+
+# Create/update using multiple hosts_groups
```

Comment:
Maybe something more explicit here to state that host matching only one of the provided host groups are not being processed.

```suggestion
# Create/update using multiple hosts_groups. Hosts present in only one of the mentioned groups will be skipped
```